src/components/drawers/MeasurementsDrawer/measurement-tables/roiManager.ts

```diff
@@ -0,0 +1,24 @@
+//@ts-nocheck
+import * as ImageJS from "image-js";
+
+export const theThing = (mask, shape) => {
```

Comment:
Are we keeping this? Not used currently.

Reply:
This is removed in graphing pr